docs/csharp/methods.md

```diff
@@ -34,7 +34,7 @@ This topic contains the following sections:
 - [Expression-bodied members](#expr)
 - [Iterators](#iterators)
 
-<a name="signatures" /a>
+<a name="signatures" />
```

Comment:
Not a void element and we're HTML5 here serving `text/html` content, so I recommend going with a closing tag `<a name="signatures"></a>` for now. We should anchor to section headings not to anchor tags. This is one that we'll hopefully permanently fix on a future pass. Just for the time being if you close it `</a>`, it should be safe.